src/Core/Users/login.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+require_once "UserInterface.php";
+
+Class login implements UserInterface{
+
+	public $_email;
+	public $_password;
+	public $_firstName;
+	public $_lastName;
+	public $_createdAt;
+	public $_updatedAt;
+	
+	public function __construct($email, $password){

```

Comment:
Lets remove the constructor for the time being. We will be a using a dependency injection container to create objects for us. Will get to the details later.